test/HotKeys/SpecifyingKeyMapUsingObjects.spec.js

```diff
@@ -8,124 +8,238 @@ import KeyCode from '../support/Key';
 import FocusableElement from '../support/FocusableElement';
 
 describe('Specifying key map using objects:', () => {
-  context('when a keydown keymap is specified as an object', () => {
```

Comment:
🐥Use the existing tests for defining a key map using an object, nest them under a more specific `context` and copy them down and adapt into another `context` that deals with defining key maps using objects with the `sequences` attribute.